backend/cesium_api/app.py

```diff
@@ -2,6 +2,7 @@
 from flask import Flask, Blueprint
 from flask_cors import CORS
 from . import DEFAULT_DATA_DIR
+from . import DEFAULT_TILES_DIR
```

Comment:
```suggestion
```

See related comment below about `DEFAULT_DATA_DIR` vs `DEFAULT_TILES_DIR`.

But make sure to remove this here.